source/includes/tutorial1/_training.html.md

```diff
@@ -0,0 +1,69 @@
+# Train Your BRAIN
+
+Now that you’ve written a curriculum of machine teaching (through your Inkling code) to connect your simulation to the Bonsai AI Engine, it’s time to prepare a new version for training. Since we previously created a new BRAIN for training, the `--brain` argument in the below commands is optional, but we’ve left it in to remind you which BRAIN the CLI is targeting.
+
+```shell
+# Push the edited files to the server
+bonsai push
+
+# Start a new BRAIN version
+bonsai train start
+```
+
+Use [`bonsai push`][1] to upload your edited Inkling file whenever you make changes (make sure you filled in the action and state schemas first or you will get an error!) to the server.
+
+> Python 2
+
+```shell
+python move_a_point_sim.py --brain=move-a-point
+```
+> Python 3
+
+```shell
+python3 move_a_point_sim.py --brain=move-a-point
+```
+
+Once you have started training mode with [`bonsai train start`][2] it's time to start running your simulator by calling Python and then the simulator file. Training will begin automatically after you connect your simulator.
+
+## View your BRAIN training status
+
+> ![Training BRAIN](../images/tutorial1-training.png)
+
+View your BRAIN's training status as it trains on the simulator by going to the BRAIN's Dashboard page on [beta.bons.ai][3]. Training move-a-point takes about a minute to get sufficient training to find the goal quickly.
+
+There is no automatic ending to training, you can train this brain for hours, but there will be diminishing returns after a few minutes because of how simplistic of a problem this is to solve. You should wait until the reward approaches and stabilizes around 18 (with the max being 20 if the AI is perfect every episode). This should only take about a minute.
+
+## Stop Training
+
+```shell
+bonsai train stop
+```
+
+Once the BRAIN has gotten to this level of performance (or sooner if you prefer), CTRL-C to disconnect the simulator, then [`bonsai train stop`][4] will end the training, and proceed to prediction.
+
+# Predict with Your BRAIN
+
+> Python 2
+
+```shell
+python move_a_point_sim.py --predict=latest
+```
+
+> Python 3
+
+```shell
+python3 move_a_point_sim.py --predict=latest
+```
+
+After your BRAIN is finished training you can use it to move to a point as quickly as it can. How well it does depends on how long you let it train! Using your BRAIN involves calling Python on your simulator file, but now in prediction mode with `--predict=latest` which will use the version of the latest training session that you just ran. You can also specify a number like `--predict=1` if you want to predict from a different version of your brain if you have trained it multiple times.
```

Comment:
perhaps remove last sentence about versions? It might lead to unneeded confusion, given the minimal support for versions today...